src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -209,6 +209,23 @@ def load_python_module(content, source=None):
         ) from e
 
 
+def load_function_from_function_path(func_path: str):
```

Comment:
Looks inappropriate to put in `tool_meta_generator`

Reply:
Let's move all codes except `gen_dynamic_list` to `https://github.com/microsoft/promptflow/blob/dfbad48467a25dc0df424a0210f08a1c74a61732/src/promptflow/promptflow/_utils/tool_utils.py`

Reply:
moved